### DISCUSSION TITLE
Xiaomi Mi Box 4 - 16-11-2024

### DISCUSSION THREAD

**Discussion by Obegg** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#721] [APK sharing]
[Xiaomi Mi Box 4 - Part 1.zip](https://github.com/user-attachments/files/17784845/Xiaomi.Mi.Box.4.-.Part.1.zip)
[Xiaomi Mi Box 4 - Part 2.zip](https://github.com/user-attachments/files/17784848/Xiaomi.Mi.Box.4.-.Part.2.zip)

Unlisted packages from `Xiaomi Mi Box 4` from today (16-11-2024) after Android Software update
@AnonymousWP @Blitz517 @Frigyes06

**Comment by AnonymousWP** [comment]
Interesting, thanks. I however don't have the time unfortunately to research all those packages, but hopefully someone does in the future.

**Reply by Obegg** [reply]
Should I open a draft PR that will add those packages but without any description, till someone will add description and etc?

**Reply by AnonymousWP** [reply]
It's okay - if you open a PR, it might get stale and may need a lot of rebases when someone's working on it. Perhaps better to wait until someone can work on it. But, thanks again! I'll keep this discussion open.

**Reply by Obegg** [reply]
@AnonymousWP I would like to upload some more .apk files from other devices (now it's over 170MB), but the size of it is too much to upload on GitHub, how else can I share it? Discord has a 10MB upload limit, GitHub has 25MB upload limit, I don't want to use Google for that, and manually splitting the zip files is too much of a hassle, when I first shared .apk files I used to send them to Blitz on Telegram, but it appears he's not active anymore, so is there someone else from the team which can use Telegram and would like to receive the apk files?

**Reply by AnonymousWP** [reply]
Hey, thanks for again for wanting to submit packages and upload some. However, I'm afraid there's not someone at the moment who's willing to analyse those packages, sorry. But hopefully someone in the future will do. Maybe @schbrns does, but he's also busy. As for storage, perhaps https://www.apkmirror.com/ will work! 🙂 It's backed by AndroidPolice and I use it regularly.
